### ISSUE TITLE
Issue with undeclared zone 

### ISSUE THREAD

**Issue by peros550** [dustindclark/homebridge-envisalink#85]
I just installed latest version and homebridge crashes. Used to work fine with same json in previous version. 

For reference, I have not declared zone three in json because I don't need to see the zone's status in homekit. 

Here is the log: 

```
[28/11/2021, 11:33:12] Error: Zone config is (undefined) for zone 3. Check config: {}
    at transformZoneStatus (/usr/local/lib/node_modules/homebridge-envisalink/src/util.ts:22:15)
    at EnvisalinkHomebridgePlatform.zoneUpdate (/usr/local/lib/node_modules/homebridge-envisalink/src/platform.ts:307:41)
    at EventEmitter.emit (events.js:315:20)
    at updatezone (/usr/local/lib/node_modules/homebridge-envisalink/node_modules/nodealarmproxy/nodealarmproxy.js:155:18)
    at Socket.<anonymous> (/usr/local/lib/node_modules/homebridge-envisalink/node_modules/nodealarmproxy/nodealarmproxy.js:278:8)
    at Socket.emit (events.js:315:20)
    at addChunk (internal/streams/readable.js:309:12)
    at readableAddChunk (internal/streams/readable.js:284:9)
    at Socket.Readable.push (internal/streams/readable.js:223:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:188:23)
```

**Comment by fifibest** [comment]
I have the same issue with non-consecutive zones, my config specifies the zoneNumber for each zone, it was working with the previous versions, but it crashes with the above error message.

**Comment by bolducd** [comment]
I have the same problem this morning after updating to 1.0.0

**Comment by anupchavda1** [comment]
I have the same issue. I have declared zones 1-12 for hard wired devices and then from zone 25 onwards for wireless zones. I am now getting the above error on zone 13 (I assume it will freak out until zones 13-24 are declared, but I don't have anything mapped to these zones). Was working fine on previous version

**Comment by marckagan** [comment]
I have the same issue.  Worked fine on previous version.

**Comment by dustindclark** [comment]
Fixed in 1.0.1. Sorry for the trouble (1.0.0 was a complete rewrite). Please re-open if you still see issues.

**Comment by peros550** [comment]
No worries man! Thank you for the quick fix
